src/commands/createContainerApp/EnvironmentVariablesListStep.ts

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { parse } from "dotenv";
+import { AzExtFsExtra, AzureWizardPromptStep } from "vscode-azureextensionui";
+import { localize } from "../../utils/localize";
+import { selectWorkspaceFile } from "../../utils/workspaceUtils";
+import { IContainerAppContext } from "./IContainerAppContext";
+
+const skipForNowLabel: string = localize('skipForNow', '$(clock) Skip for now');
+
+export class EnvironmentVariablesListStep extends AzureWizardPromptStep<IContainerAppContext> {
+    public async prompt(context: IContainerAppContext): Promise<void> {
+        const input = await context.ui.showQuickPick([{ label: localize('set', 'Set with environment variable file') }, { label: skipForNowLabel }],
```

Comment:
Should this be “environment variable file” or “environment variables file”? At first glance I thought it might be “variables” because it tells me I can have more than 1 variable in there but not sure about grammatically.

Reply:
I think you're correct, but it might actually be better to just go with `.env file` ala https://www.ibm.com/docs/en/aix/7.2?topic=files-env-file